src/registry/SsiAuthoritySchemaRegistry.Service/Program.cs

```diff
@@ -0,0 +1,48 @@
+/********************************************************************************
+ * Copyright (c) 2024 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+using Org.Eclipse.TractusX.Portal.Backend.Framework.Web;
+using Org.Eclipse.TractusX.SsiAuthoritySchemaRegistry.DbAccess.DependencyInjection;
+using Org.Eclipse.TractusX.SsiAuthoritySchemaRegistry.Service.Controllers;
+using System.Text.Json.Serialization;
+
+const string Version = "v1";
+
+WebApplicationBuildRunner
+    .BuildAndRunWebApplication<Program>(args, "registry", Version, ".Registry",
```

Comment:
as await is supported in toplevel program.cs BuildAndRunWebApplication should be refactored as async and awaited here. (this saves a thread that otherwiese synchronously waits for program.cs to end).

Reply:
will adjust it as soon as https://github.com/eclipse-tractusx/portal-backend/pull/786 is merged and the new packages are available.

Reply:
adjusted